tools/automation/cache/waitForCache.rc

```diff
@@ -0,0 +1,13 @@
+<ruby>
```

Comment:
Small nit pick, I think the file naming convention is to stick with snake case 🐍

Reply:
I had originally chosen to used different convention to signify the tool (`update_module_cache.sh`) vs support files for the tool (`waitForCache.rc` && `buildNewCache.sh`).

I am not strongly tied to the idea, just looking to differentiate what user should interact with here.